CHANGELOG.md

```diff
@@ -95,4 +95,393 @@ release.
 - Mark service.version as stable.
   ([#106](https://github.com/open-telemetry/semantic-conventions/pull/106))
 - Mark initial set of HTTP semantic conventions as frozen
-  ([#105](https://github.com/open-telemetry/semantic-conventions/pull/105))  
+  ([#105](https://github.com/open-telemetry/semantic-conventions/pull/105))
+
+## v1.20.0 (2023-04-07)
+
+- Clarify that attribute requirement levels apply to the instrumentation library
```

Comment:
```suggestion
## v1.20.0 (2023-04-07)

This and earlier versions were released as part of [the Specification](https://github.com/open-telemetry/opentelemetry-specification/).

- Clarify that attribute requirement levels apply to the instrumentation library
```

Reply:
We can refactor that later, I like this suggestion though.